static/docs/commands-reference/fetch.md

```diff
@@ -1,62 +1,287 @@
+[example](/doc/get-started/example-pipeline):
+
+```shell
+    .
+    ├── data
+    │   └── data.xml.dvc
+    ├── evaluate.dvc
+    ├── featurize.dvc
+    ├── prepare.dvc
+    ├── train.dvc
+    └── src
+        └── <code files here>
+```
+
+We have these tags in the repository that represent different iterations of
+solving the problem:
+
+```shell
+    $ git tag
+
+    baseline     <- first simple version of the model
+    bigram       <- use bigrams to improve the model
+```
+
+## Examples: Default behavior
+
+This project comes with a predefined S3 [remote
+storage](https://man.dvc.org/remote). We can now just run `dvc fetch`
+that will download the most recent `model.pkl`, `data.xml`, and other files that
+are under DVC control into our local cache:
+
+```shell
+    $ ldvc status -c    <- compares local cache vs default remote
 
-```dvc
     $ dvc fetch
+    ...
+    (2/6): [##############################] 100% data/features/test.pkl
+    (3/6): [##############################] 100% model.pkl
+    (4/6): [##############################] 100% data/features/train.pkl
+    ...
+
+    $ tree .dvc
+    .dvc
+    ├── cache           <- dir .dvc/cache was created and populated
+    │   ├── 38
+    │   │   └── 63d0e317dee0a55c4e59d2ec0eef33
+    │   ├── 42
+    │   │   └── c7025fc0edeb174069280d17add2d4.dir
+    │   ├── ...
+    ├── config
+    ├── ...
+```
+
+As seen above, used without arguments, `dvc fetch` downloads all assets needed
+by all DVC files in∑ the current branch, including for directories. The checksums
+`3863d0e317dee0a55c4e59d2ec0eef33` and `42c7025fc0edeb174069280d17add2d4`
+correspond to the `model.pkl` file and `data/features/` directory, respectively.
+
+```shell
+    $ dvc checkout      <- links files from local cache to workspace
+    Checking out '{'scheme': 'local', 'path': '.../example-get-started/model.pkl'}' with cache '3863d0e317dee0a55c4e59d2ec0eef33'.
+    Checking out '{'scheme': 'local', 'path': '.../example-get-started/data/features'}' with cache '42c7025fc0edeb174069280d17add2d4.dir'.
+    ...
+```
+
+## Examples: Specific stages
+
+Fetch only outputs of a specific stage bt specifying its DVC file (target
+stage):
 
-    (1/8): [#################################] 100% images/0001.jpg
-    (2/8): [#################################] 100% images/0002.jpg
-    (3/8): [#################################] 100% images/0003.jpg
-    (4/8): [#################################] 100% images/0004.jpg
-    (5/8): [#################################] 100% images/0005.jpg
-    (6/8): [#################################] 100% images/0006.jpg
-    (7/8): [#################################] 100% images/0007.jpg
-    (8/8): [#################################] 100% model.pkl
+```shell
+    $ dvc fetch prepare.dvc
+    ...
+    (1/2): [##############################] 100% data/prepared/test.tsv
+    (2/2): [##############################] 100% data/prepared/train.tsv
+
+    $ tree .dvc/cache      
+.dvc/cache
```

Comment:
indent

Reply:
why there are two directories in cache btw? Should we run `dvc gc` before lllustrating the command?

Reply:
Indented. The 2 directories are created by dvc when I run `dvc status` or fetch or pull. I guess they correspond to data/features and data/prepared? `dvc gc` tells me "No unused local cache to remove." and both dirs are left there.